python/huggingfaceserver/tests/vllm_mock_outputs.py

```diff
@@ -20,7 +20,7 @@
 opt_chat_cmpl_chunks = [
     RequestOutput(
         request_id="cmpl-d771287a234c44498e345f0a429d6691-0",
-        prompt="You are a friendly chatbot who always responds in the style of a pirate</s>How many helicopters can a human eat in one sitting?</s>",
+        prompt=None,
```

Comment:
So the RequestOutput no longer returns the decoded token right?

Reply:
Yes. Previously we can pass both prompt and prompt token id to the generate endpoint. But, Now we can only pass either of them. Since we are passing prompt token ids to generate endpoint, we won't get decoded prompt in RequestOutput